### ISSUE TITLE
NTR: Uterine cervix biopsy: papilloma  (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9448]
New term request Uterine cervix biopsy: papilloma  (elthat_211226184433): Papillomatosis; uterus  UMLS: 5208237

**Comment by MickeySegal** [comment]
That seems like the same thing, we got it from https://www.omim.org/entry/193670.  But https://hpo.jax.org/app/browse/term/HP:0031021 has only a child term for oropharyngeal.

**Comment by pnrobinson** [comment]
@MickeySegal 
added as HP:4000153
label:Cervical squamous cell papilloma
def: A benign tumor arising from the epithelial surface of the cervix and usually caused by Human Papillomavirus infection.
